From 7fa052cef9680e30e07a74423c440a2b79e81916 Mon Sep 17 00:00:00 2001
From: Yu Watanabe <watanabe.yu+github@gmail.com>
Date: Fri, 1 Sep 2017 17:03:01 +0900
Subject: [PATCH] systemd-mount: fix that wrong argument is used for
 arg_mount_what

(cherry picked from commit 4185da7c4daf32bd9b5809fd6de99e52cb903bba)
---
 src/mount/mount-tool.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/mount/mount-tool.c b/src/mount/mount-tool.c
index ed6578d540..10bb5dda63 100644
--- a/src/mount/mount-tool.c
+++ b/src/mount/mount-tool.c
@@ -344,9 +344,8 @@ static int parse_argv(int argc, char *argv[]) {
                         arg_mount_what = canonicalize_file_name(p);
                         if (!arg_mount_what)
                                 return log_error_errno(errno, "Failed to canonicalize path: %m");
-
                 } else {
-                        arg_mount_what = strdup(argv[optind+1]);
+                        arg_mount_what = strdup(argv[optind]);
                         if (!arg_mount_what)
                                 return log_oom();
 
